<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GearCity ModTools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="CityEditorWidgetContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QGroupBox" name="MainMenuGroupBoxContainer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1261</width>
       <height>671</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="4">
       <widget class="QPushButton" name="Button_PreMadeVehicleLoader">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Premade Vehicle Loader</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Vintage car_#222222_64px.png</normaloff>:/.png/Vintage car_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="Button_MapDataTools">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Map Data Tools</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/globe-icon.png</normaloff>:/.png/globe-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="Button_LogoEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Player Logo Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Paint_#222222_64px.png</normaloff>:/.png/Paint_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="Button_NewsPaperEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Newspaper Article Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/newspaper-icon.png</normaloff>:/.png/newspaper-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="Line" name="line_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="Button_LocalizationEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Localization Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Edit text_#222222_64px.png</normaloff>:/.png/Edit text_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="Button_TurnEventsEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Turn Events Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Calendar with circles_#222222_64px.png</normaloff>:/.png/Calendar with circles_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="Line" name="line_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="Button_CityEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>City Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Block of flats_#222222_64px.png</normaloff>:/.png/Block of flats_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="Line" name="line_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Line" name="line_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="Button_DYKEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Did You Know Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Dialog_#222222_64px.png</normaloff>:/.png/Dialog_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="Button_MusicEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Music Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Music note_#222222_64px.png</normaloff>:/.png/Music note_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="Button_ScenarioDataTools">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Scenario Data Tools</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/folder-games-icon.png</normaloff>:/.png/folder-games-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="Button_Racing">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Racing Series Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/X_#222222_64px.png</normaloff>:/.png/X_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="Button_MagazineEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Magazine Review Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/Book with writing_#222222_64px.png</normaloff>:/.png/Book with writing_#222222_64px.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Button_ModDataTools">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Mod Data Tools</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/make-icon.png</normaloff>:/.png/make-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="Button_AIEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>AI Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/chess-horse-icon.png</normaloff>:/.png/chess-horse-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="Button_ComponentEditor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Components Editor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/.png/tire-icon.png</normaloff>:/.png/tire-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="AIEditorWidgetContainer" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="MapDataToolWidgetContainer_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="MapDataToolWidgetContainer_5" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="MapDataToolWidgetContainer_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="MapDataToolWidgetContainer_7" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="MapDataToolWidgetContainer_8" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="MapDataToolWidgetContainer_9" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="TurnEventEditorWidgetContainer" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="MapDataToolWidgetContainer" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>CityEditorWidgetContainer</zorder>
   <zorder>AIEditorWidgetContainer</zorder>
   <zorder>MapDataToolWidgetContainer_4</zorder>
   <zorder>MapDataToolWidgetContainer_5</zorder>
   <zorder>MapDataToolWidgetContainer_6</zorder>
   <zorder>MapDataToolWidgetContainer_7</zorder>
   <zorder>MapDataToolWidgetContainer_8</zorder>
   <zorder>MapDataToolWidgetContainer_9</zorder>
   <zorder>TurnEventEditorWidgetContainer</zorder>
   <zorder>MapDataToolWidgetContainer</zorder>
   <zorder>MainMenuGroupBoxContainer</zorder>
  </widget>
  <action name="actionScenario">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDelete_2">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionMapNew">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionModNew">
   <property name="text">
    <string>Mod</string>
   </property>
  </action>
  <action name="actionScenarioNew">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionMapOpen">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionModOpen">
   <property name="text">
    <string>Mod</string>
   </property>
  </action>
  <action name="actionScenarioOpen">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionMapDelete">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionModDelete">
   <property name="text">
    <string>Mod</string>
   </property>
  </action>
  <action name="actionScenarioDelete">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp_Page">
   <property name="text">
    <string>Help Page</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFinished_Map">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionFinished_Mod">
   <property name="text">
    <string>Finished Mod</string>
   </property>
  </action>
  <action name="actionFinished_Scenario">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionMapImport">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionModImport">
   <property name="text">
    <string>Mod</string>
   </property>
  </action>
  <action name="actionScenarioImport">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
